cms/topics/tests/test_models.py

```diff
@@ -35,10 +35,12 @@ def setUpTestData(cls):
         )
 
         cls.topic_page.featured_series = cls.article_series
-        cls.topic_page.save()
+        cls.topic_page.save(update_fields=["featured_series"])
 
-        cls.methodology = MethodologyPageFactory(parent=cls.topic_page, publication_date=datetime(2024, 6, 1))
-        cls.another_methodology = MethodologyPageFactory(parent=cls.topic_page, publication_date=datetime(2024, 11, 1))
+        cls.methodology = MethodologyPageFactory(parent__parent=cls.topic_page, publication_date=datetime(2024, 6, 1))
+        cls.another_methodology = MethodologyPageFactory(
+            parent=cls.methodology.get_parent(), publication_date=datetime(2024, 11, 1)
+        )
```

Comment:
Question: Why do you do `parent__parent` here `cls.methodology = MethodologyPageFactory(parent__parent=cls.topic_page, publication_date=datetime(2024, 6, 1))`

Then the next line `parent` here `cls.another_methodology = MethodologyPageFactory(parent=cls.methodology.get_parent(), publication_date=datetime(2024, 11, 1)`

Reply:
The methodology page parent is now a methodology index page, so `parent__parent=cls.topic_page` ensure that `cls.methodology` is created under a methodology index page that is created under `cls.topic`.
As for `cls.another_methodology`, we want to use the freshly created methodology index page as a parent, which is `cls.methodology.get_parent()`

Reply:
Thanks Dan, make sense!